codespell_lib/data/dictionary.txt

```diff
@@ -12108,6 +12111,7 @@ emphsis->emphasis
 emphysyma->emphysema
 empirial->empirical, imperial,
 empiricaly->empirically
+emply->empty, imply,
```

Comment:
Personally, I would suggest `employ` first. Typo for `empty` and `imply` are rare as the keys are far apart.
```
emply->employ, empty, imply,
emplyed->employed
emplyee->employee
emplyees->employees
emplyer->employer
emplyers->employers
emplying->employing
emplyment->employment
emplyments->employments
```

Reply:
f5bbcfd